src/phonebook/forms.py

```diff
@@ -0,0 +1,57 @@
+import re
+import secrets
+
+from django import forms
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.urls import reverse
+
+from django.core.exceptions import ValidationError
+
+from .models import DectRegistration
+from .dectutils import DectUtils
+
+dectutil = DectUtils()
+
+
+class DectRegistrationForm(forms.ModelForm):
+    ipei_help_text = """
+    Optional: Enter your IPEI (03562 0900847) or IPUI (00DEADBEEF)<br>
+    Entering this will enable you to register your handset directly when you arrive
+    """
+    ipei = forms.CharField(
+        max_length=13,
+        help_text=ipei_help_text,
+        required=False,
+        label="IPEI/IPUI",
+    )
```

Comment:
```suggestion
class DectRegistrationForm(forms.ModelForm):
    ipei = forms.CharField(
        max_length=13,
        help_text="Optional: Enter your IPEI (03562 0900847) or IPUI (00DEADBEEF)<br>Entering this will enable you to use your handset immediately when you arrive",
        required=False,
        label="IPEI/IPUI",
    )
```